ui/src/pages/Input/InputPage.tsx

```diff
@@ -36,50 +43,98 @@ const Row = styled(ColumnLayout.Row)`
     }
 `;
 
-function InputPage() {
-    const [entity, setEntity] = useState({ open: false });
-    const unifiedConfigs = getUnifiedConfigs();
-    const { services, title, table, description, subDescription } = unifiedConfigs.pages.inputs;
+// Define the types based on the Zod schemas
+type InputsPageRegular = z.infer<typeof InputsPageRegular>;
+type InputsPageTableSchema = z.infer<typeof InputsPageTableSchema>;
 
-    // check if the tabs feature is enabled or not.
-    const isTabs = !table;
+type InputsPage = {
+    type: 'regular' | 'table';
+} & (InputsPageRegular | InputsPageTableSchema);
```

Comment:
It allows the creation of objects that have both 'regular' and 'table' properties, which doesn't make logical sense and could lead to runtime errors. TypeScript might not be able to infer the correct properties based on the 'type' field, requiring more type assertions or checks in your code.